run_page/generator/db.py

```diff
@@ -29,6 +29,7 @@ def randomword():
     "distance",
     "moving_time",
     "type",
+    "subtype",
```

Comment:
This new column is not present in the SQLite file if the user syncs before this pull request. Users may face difficulties adding this column manually, or else syncing will fail. Perhaps you should add instructions in the readme or include a script to assist users in doing this easily, as seen in this [ref1](https://github.com/ben-29/workouts_page?tab=readme-ov-file#note3-20240819-added-elevation-gain-field-if-you-forked-the-project-before-this-update-please-run-the-following-command) [ref2](https://github.com/ben-29/workouts_page/blob/master/run_page/db_updater.py). There might be a simpler solution that I'm unaware of.

Reply:
The db checks whether the new column exists after commit https://github.com/yihong0618/running_page/pull/733/commits/c86d04042395dbd5187a8c98186962bb78128fd9. If any column defined in `class Activity` does not exist, it will add a new column in data.db.